finality-provider/config/config.go

```diff
@@ -18,26 +18,24 @@ import (
 )
 
 const (
-	defaultChainType               = "babylon"
-	defaultLogLevel                = zapcore.InfoLevel
-	defaultLogDirname              = "logs"
-	defaultLogFilename             = "fpd.log"
-	defaultFinalityProviderKeyName = "finality-provider"
-	DefaultRPCPort                 = 12581
-	defaultConfigFileName          = "fpd.conf"
-	defaultNumPubRand              = 70000 // support running of 1 week with block production time as 10s
-	defaultNumPubRandMax           = 100000
-	defaultMinRandHeightGap        = 35000
-	defaultStatusUpdateInterval    = 20 * time.Second
-	defaultRandomInterval          = 30 * time.Second
-	defaultSubmitRetryInterval     = 1 * time.Second
-	defaultFastSyncInterval        = 10 * time.Second
-	defaultSyncFpStatusInterval    = 30 * time.Second
-	defaultFastSyncLimit           = 10
-	defaultFastSyncGap             = 3
-	defaultMaxSubmissionRetries    = 20
-	defaultBitcoinNetwork          = "signet"
-	defaultDataDirname             = "data"
+	defaultChainType                   = "babylon"
+	defaultLogLevel                    = zapcore.InfoLevel
+	defaultLogDirname                  = "logs"
+	defaultLogFilename                 = "fpd.log"
+	defaultFinalityProviderKeyName     = "finality-provider"
+	DefaultRPCPort                     = 12581
+	defaultConfigFileName              = "fpd.conf"
+	defaultNumPubRand                  = 70000 // support running of 1 week with block production time as 10s
+	defaultNumPubRandMax               = 100000
+	defaultMinRandHeightGap            = 35000
+	defaultStatusUpdateInterval        = 20 * time.Second
+	defaultRandomInterval              = 30 * time.Second
+	defaultSubmitRetryInterval         = 1 * time.Second
+	defaultSyncFpStatusInterval        = 30 * time.Second
+	defaultSignatureSubmissionInterval = 1 * time.Second
+	defaultMaxSubmissionRetries        = 20
+	defaultBitcoinNetwork              = "signet"
+	defaultDataDirname                 = "data"
```

Comment:
Shouldn't we add a config for batch processing size? what's the current code behavior if it lags way behind?

Reply:
Good point. Fixed.

Reply:
If the fp lags behind, the poller should quickly poll blocks (that's why we should have a short polling interval). Then the fp will do batch submission